src/ClusterManager/job.py

```diff
@@ -219,6 +234,35 @@ def _get_cluster_config(self, key):
             return self.cluster[key]
         return None
 
+    #only add blobfuse since we only support one plugin right now
+    def get_plugins(self):
+        '''
+        Plugin example:
+            "blobfuse": {
+                "enabled":true,
+                "name":"blobfuse",
+                "accountName":"YWRtaW4=",
+                "accountKey":"MWYyZDFlMmU2N2Rm",
+                "containerName":"blobContainer",
+                "mountPath":"/data",
```

Comment:
Can we use another path instead of `/data` as an example? `/data` is currently the cluster-wide shared data path.